projects/rocprofiler-systems/source/lib/core/config.cpp

```diff
@@ -1125,7 +1135,7 @@ configure_mode_settings(const std::shared_ptr<settings>& _config)
     else if(get_mode() == Mode::Causal)
     {
         _set("ROCPROFSYS_USE_CAUSAL", true);
-        _set("ROCPROFSYS_TRACE", false);
+        _set("ROCPROFSYS_TRACE_LEGACY", false);
```

Comment:
I think we need to disable trace in "causal" mode too.

```suggestion
        _set("ROCPROFSYS_TRACE_LEGACY", false);
        _set("ROCPROFSYS_TRACE", false);
```